tox.ini

```diff
@@ -57,9 +57,9 @@ deps =
     dask[array,diagnostics]>=2021.04.1
 depends = begin
 commands =
-    # don't do doctests in rst files due to lack of way to specify optional
```

Comment:
Is there a reason you took this out here? It wasn't mentioned in the description and it seems unrelated to load times.

Reply:
I can't recall why I did that. Let's try to revert this part and see if anything fails.

Reply:
welp, nothing breaks. I'll just clean up the history